lib/parser.js

```diff
@@ -236,6 +240,30 @@ module.exports = function parser(identifier, string) {
     expect(Token.Type.TERMINATOR)
   }
 
+  // Parses oneof: oneof oneofName {protoType fieldName = 1;}
+  function parseOneof(parent) {
+    var oneofName = expect(Token.Type.WORD).content
```

Comment:
does protoc give an error if you have two oneofs with the same name in the same field?